src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,18 +3,55 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_LENGTH = 10;
+    private final K[] keyArray = (K[]) new Object[ARRAY_LENGTH];
+    private final V[] valueArray = (V[]) new Object[ARRAY_LENGTH];
```

Comment:
it's better initialise arrays in constructor
```suggestion
    private final K[] keys = (K[]) new Object[ARRAY_LENGTH];
    private final V[] values = (V[]) new Object[ARRAY_LENGTH];
```